layouts/partials/topbar.html

```diff
@@ -13,6 +13,7 @@
         <li><a href="https://github.com/libp2p/specs" target="_blank">Specifications</a></li>
         <li><a href="https://docs.libp2p.io" target="_blank">Documentation</a></li>
         <li><a href="/#community">Community</a></li>
+        <li><a href="https://connectivity.libp2p.io" target="_blank">Universal Connectivity</a></li>
```

Comment:
```suggestion
        <li><a href="https://connectivity.libp2p.io" target="_blank">Connectivity</a></li>
```
Let's remove Universal
Preview looks cramped
<img width="562" alt="image" src="https://user-images.githubusercontent.com/50885601/198342688-c3dd71b1-bf71-446e-9494-bfaf6d473f84.png">